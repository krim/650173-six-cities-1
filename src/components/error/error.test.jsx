import React from 'react';
import renderer from 'react-test-renderer';

import Error from './error';

describe(`Error`, () => {
  it(`renders component correctly`, () => {
    const tree = renderer.
      create(<Error message={`text`}/>).
      toJSON();

    expect(tree).toMatchSnapshot();
  });
});
