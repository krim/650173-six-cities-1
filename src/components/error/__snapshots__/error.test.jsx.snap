// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders component correctly 1`] = `
<div
  style={
    Object {
      "color": "#FF0000",
      "marginBottom": "1em",
    }
  }
>
  text
</div>
`;
